tests/testthat/test-dupree_package_integration.R

```diff
@@ -1,15 +1,18 @@
-context("Integration tests for `dupree_package()`")
+context("Integration tests for `dupree_package`")
 
 test_that("`dupree_package` results only include files from <my_pkg>/R/", {
   # the test-package "anRpackage" contains
   # - ./R/anRpackage-internal.R
   # - and ./inst/dir1/R/dont_dup_me.R
-  # - the latter should not be included by dupree_package() by default
+  # - the latter should not be included by `dupree_package` by default
+  expect_is(
+    dupree_package(file.path("testdata", "anRpackage")), "dups"
+  )
   expect_silent(
-    dupree_package(file.path("testdata", "anRpackage"))
+    as.data.frame(dupree_package(file.path("testdata", "anRpackage")))
```

Comment:
Similarly, this should be `expect_silent(dupree_package(args))`; the data.frame conversion is irrelevant to this test